api/api/clubs/tests.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+
+User = get_user_model()
+
+
+class ListClubsViewTest(APITestCase):
+    """Tests ListClubView for getting the list of clubs."""
+
+    def testListClubs_noClubExistsInDb_returnsEmptyList(self):
```

Comment:
mark these tests with `@python.markers.django_db`, it ensures that a clean db is used for each test and cleans up after the test finishes: [reference](https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest.mark.django_db)

Look at tests in other files and refactor them as well.

Reply:
> mark these tests with `@python.markers.django_db`, it ensures that a clean db is used for each test and cleans up after the test finishes: [reference](https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest.mark.django_db)
> 
> Look at tests in other files and refactor them as well.

Done!